ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -604,3 +604,72 @@ literals:
 
 # Added 4/12/20
 - supply_air_damper_percentage_sensor
+
+# Added 9/8/20 for SG-SIN-MBC2
+- chilled_return_water_flowrate_sensor
+- device_fault_status
+- fire_alarm_status
+- fire_fault_status
+- makeup_water_high_tank_percentage_status
+- makeup_water_low_tank_percentage_status
+- max_cooling_air_flowrate_setpoint
+- min_cooling_air_flowrate_setpoint
+- pump_fault_status
+- pump_run_status
+- recovery_alarm_command
+- schedule_command
+- photocatalyst_filter_run_status
+- water_high_tank_alarm_status
+- water_leak_alarm_status
+- water_leak_detection_fault_status
+- zone_command
+# - automatic_mode
+# - ballast_status
+
+# Added 9/14/20 for TW-NTC-TPKD
+- condensing_water_valve_percentage_sensor
+- primary_speed_percentage_command
+- primary_speed_percentage_sensor
+- secondary_speed_percentage_command
+- secondary_speed_percentage_sensor
+- gas_leak_detection_alarm_status
+- air_discharge_compressor_pressure_sensor
+- mixed_air_discharge_compressor_pressure_sensor
+- hot_water_supply_temperature_sensor
+- hot_water_return_temperature_sensor
+- mixing_hot_water_supply_temperature_sensor
+- mixing_hot_water_return_temperature_sensor
+- hot_water_circulation_supply_temperature_sensor
+- hot_water_circulation_return_temperature_sensor
+- air_dryer_dewpoint_temperature_sensor
+- air_dryer_pressure_sensor
+- air_dryer_flowrate_sensor
+- zone_dimmer_percentage_command
+- zone_scene_command
+- zone_scene_dimmer_percentage_command
+- air_differential_pressure_status
+- air_differential_pressure_sensor
+- air_static_pressure_sensor
+- high_heater_power_command
+- low_heater_power_command
+- medium_heater_power_command
```

Comment:
same as above

Reply:
This is a electrical heater on/off signal for medium power consume, can you suggest me a suitable point naming?

Reply:
amend to "medium_heater_run_command"